packages/relayer/src/msgs.rs

```diff
@@ -0,0 +1,61 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, Binary};
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub relayer_pubkey: Binary,
+    pub relayer_address: String,
```

Comment:
Would be useful to add a comment explaining what this refers to. Since it's strange including a relayer address while instantiating a relayer contract